package.json

```diff
@@ -1,24 +1,32 @@
 {
   "name": "reddit_viewer",
   "version": "0.1.0",
-  "main": "dist/app.js",
+  "main": "dist",
   "private": true,
   "scripts": {
-    "test": "tslint --project ./tsconfig.json",
+    "test": "npm run build && npm run lint && npm run mocha",
+    "lint": "tslint --project . -t verbose",
+    "mocha": "mocha --require ts-node/register ./test/*.test.*",
```

Comment:
This project doesn't have unit tests. It would be nice to introduce some, but until then do not introduce mocha scripts and dependency